gen3/docs/gen3-resources/user-guide/search.md

```diff
@@ -2,14 +2,33 @@
+Various levels of data access can be configured in a Gen3 data commons using the Gen3 Framework Services. If open access data is hosted, a data commons can be configured to allow anonymous access to data, which means users can explore data without logging in. This is the case for the [Gen3 Data Hub][Gen3 Data Hub].
 
 In cases where data is controlled access, typically external users will receive instructions on how to access data and may be required to sign a DUA (Data Use Agreement) legal document.
 
 The following sections provide details on how to explore and access data from within the data commons website and from the command-line by sending requests to the Gen3 open APIs.
 
+[//]: # (Alex: I would like to see this ordered like this Data Discovery -> Data Access Approval - if controlled data -> Data Exploration. I would like to promote using the Gen3 Discovery page as the initial place to find data as that's it's intended purpose.)
+
+[//]: # (Alex: I started thinking through user "questions" and relating them to particular groups of endpoints in our API and thinking through how users would expect to move between parts of the product. It's old and maybe needs some dust removed, but it was done while thinking through the idea of the Data Lake. The relevant section of the feature doc is here: https://docs.google.com/document/d/1UjQjvUuasmfe_5iaEfqjFzLTHsC-8EXXNYCSyP0-k_s/edit?tab=t.0#heading=h.xp6x9p5szrmw)
+
 ## Searching for Data from the Data Portal
-The Gen3 software stack offers a data portal service that creates a website with graphical tools for performing the basic functionality of a data commons, like browsing data in projects, building patient cohorts across projects, downloading metadata or data files for cohorts, and building database queries. The data portal relies on the same [API queries][API instructions] that you can explore directly if you prefer.
+Gen3 offers a data portal service that creates a website with graphical tools for performing the basic functionality of a data commons, like browsing data in projects, building patient cohorts across projects, downloading metadata or data files for cohorts, and building database queries.
+
+While this may vary from system to system, to find data in a data commons you can follow a general workflow of 1) Discover data in a mesh or commons using the Discovery Page, 2) Request Access to data using system specific solution, 3) Explore files in the Exploration Page, and 4) Export data to workspace or download locally depending on the requirements of your system.  Instead of using the Discovery and Exploration pages you could instead use the [API][API instructions] for locating data of interest.

```

Comment:
```suggestion
While this may vary from system to system, to find data in a data commons you can follow a general workflow of:

1. Discover data in a mesh or commons using the Discovery Page
2. Request Access to data using system specific solution
3. Explore files in the Exploration Page
4. Export data to workspace or download locally depending on the requirements of your system.

Instead of using the Discovery and Exploration pages you could instead use the [API][API instructions] for locating data of interest.

```